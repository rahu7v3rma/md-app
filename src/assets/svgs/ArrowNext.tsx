import React, { FunctionComponent } from 'react';
import { Path, Svg } from 'react-native-svg';

import { Colors } from '@/theme/colors';

type Props = {
    width?: number;
    height?: number;
    color?: string;
};

const ArrowNext: FunctionComponent<Props> = ({
    width = 24,
    height = 24,
    color = Colors.text.gray
}: Props) => {
    return (
        <Svg width={width} height={height} viewBox="0 0 24 24" fill="none">
            <Path
                d="M9.5 7L14.5 12L9.5 17"
                stroke={color}
                strokeWidth={2}
                strokeLinecap="round"
                strokeLinejoin="round"
            />
        </Svg>
    );
};

export default ArrowNext;
