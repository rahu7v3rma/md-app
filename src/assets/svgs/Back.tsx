import React, { FunctionComponent } from 'react';
import { StyleSheet } from 'react-native';
import { Path, Svg } from 'react-native-svg';

type Props = {
    width?: number;
    height?: number;
};

const BackIcon: FunctionComponent<Props> = ({
    width = 8,
    height = 14
}: Props) => {
    return (
        <Svg
            width={width}
            height={height}
            style={style.root}
            viewBox="0 0 8 14"
            fill="none"
        >
            <Path
                d="M5.97999 0.319993L2.76999 3.52999L0.79999 5.48999C-0.0300097 6.31999 -0.0300097 7.66999 0.79999 8.49999L5.97999 13.68C6.65999 14.36 7.81999 13.87 7.81999 12.92V7.30999V1.07999C7.81999 0.119992 6.65999 -0.360007 5.97999 0.319993Z"
                fill="#271A51"
            />
        </Svg>
    );
};

const style = StyleSheet.create({
    root: {
        width: 'auto',
        height: 'auto'
    }
});

export default BackIcon;
