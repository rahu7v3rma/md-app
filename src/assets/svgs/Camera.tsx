import React, { FunctionComponent } from 'react';
import { StyleSheet } from 'react-native';
import { Mask, Path, Svg } from 'react-native-svg';

import { Colors } from '@/theme/colors';

type Props = {
    width?: number;
    height?: number;
};

const Camera: FunctionComponent<Props> = ({
    width = 20,
    height = 20
}: Props) => {
    return (
        <Svg
            width={width}
            height={height}
            style={style.root}
            viewBox="0 0 20 20"
            fill="none"
        >
            <Mask id="path-1-inside-1_1_564" fill={'white'} />
            <Path d="M11.7667 3.33332L13.2917 4.99999H16.6667V15H3.33333V4.99999H6.70833L8.23333 3.33332H11.7667ZM12.5 1.66666H7.5L5.975 3.33332H3.33333C2.41666 3.33332 1.66666 4.08332 1.66666 4.99999V15C1.66666 15.9167 2.41666 16.6667 3.33333 16.6667H16.6667C17.5833 16.6667 18.3333 15.9167 18.3333 15V4.99999C18.3333 4.08332 17.5833 3.33332 16.6667 3.33332H14.025L12.5 1.66666ZM10 7.49999C11.375 7.49999 12.5 8.62499 12.5 9.99999C12.5 11.375 11.375 12.5 10 12.5C8.625 12.5 7.5 11.375 7.5 9.99999C7.5 8.62499 8.625 7.49999 10 7.49999ZM10 5.83332C7.7 5.83332 5.83333 7.69999 5.83333 9.99999C5.83333 12.3 7.7 14.1667 10 14.1667C12.3 14.1667 14.1667 12.3 14.1667 9.99999C14.1667 7.69999 12.3 5.83332 10 5.83332Z" />

            <Path
                d="M11.7667 3.33332L13.2917 4.99999H16.6667V15H3.33333V4.99999H6.70833L8.23333 3.33332H11.7667ZM12.5 1.66666H7.5L5.975 3.33332H3.33333C2.41666 3.33332 1.66666 4.08332 1.66666 4.99999V15C1.66666 15.9167 2.41666 16.6667 3.33333 16.6667H16.6667C17.5833 16.6667 18.3333 15.9167 18.3333 15V4.99999C18.3333 4.08332 17.5833 3.33332 16.6667 3.33332H14.025L12.5 1.66666ZM10 7.49999C11.375 7.49999 12.5 8.62499 12.5 9.99999C12.5 11.375 11.375 12.5 10 12.5C8.625 12.5 7.5 11.375 7.5 9.99999C7.5 8.62499 8.625 7.49999 10 7.49999ZM10 5.83332C7.7 5.83332 5.83333 7.69999 5.83333 9.99999C5.83333 12.3 7.7 14.1667 10 14.1667C12.3 14.1667 14.1667 12.3 14.1667 9.99999C14.1667 7.69999 12.3 5.83332 10 5.83332Z"
                fill={Colors.theme.primary}
            />
            <Path
                d="M13.2917 4.99999L13.1441 5.135L13.2036 5.19999H13.2917V4.99999ZM16.6667 4.99999H16.8667V4.79999H16.6667V4.99999ZM16.6667 15V15.2H16.8667V15H16.6667ZM3.33333 15H3.13333V15.2H3.33333V15ZM3.33333 4.99999V4.79999H3.13333V4.99999H3.33333ZM6.70833 4.99999V5.19999H6.79642L6.85588 5.135L6.70833 4.99999ZM8.23333 3.33332V3.13332H8.14524L8.08578 3.19831L8.23333 3.33332ZM12.5 1.66666L12.6476 1.53165L12.5881 1.46666H12.5V1.66666ZM7.5 1.66666V1.46666H7.41191L7.35244 1.53165L7.5 1.66666ZM5.975 3.33332V3.53332H6.06309L6.12255 3.46833L5.975 3.33332ZM14.025 3.33332L13.8774 3.46833L13.9369 3.53332H14.025V3.33332ZM11.6191 3.46833L13.1441 5.135L13.4392 4.86498L11.9142 3.19831L11.6191 3.46833ZM13.2917 5.19999H16.6667V4.79999H13.2917V5.19999ZM16.4667 4.99999V15H16.8667V4.99999H16.4667ZM16.6667 14.8H3.33333V15.2H16.6667V14.8ZM3.53333 15V4.99999H3.13333V15H3.53333ZM3.33333 5.19999H6.70833V4.79999H3.33333V5.19999ZM6.85588 5.135L8.38088 3.46833L8.08578 3.19831L6.56078 4.86498L6.85588 5.135ZM8.23333 3.53332H11.7667V3.13332H8.23333V3.53332ZM12.5 1.46666H7.5V1.86666H12.5V1.46666ZM7.35244 1.53165L5.82744 3.19831L6.12255 3.46833L7.64755 1.80167L7.35244 1.53165ZM5.975 3.13332H3.33333V3.53332H5.975V3.13332ZM3.33333 3.13332C2.30621 3.13332 1.46666 3.97287 1.46666 4.99999H1.86666C1.86666 4.19378 2.52712 3.53332 3.33333 3.53332V3.13332ZM1.46666 4.99999V15H1.86666V4.99999H1.46666ZM1.46666 15C1.46666 16.0271 2.30621 16.8667 3.33333 16.8667V16.4667C2.52712 16.4667 1.86666 15.8062 1.86666 15H1.46666ZM3.33333 16.8667H16.6667V16.4667H3.33333V16.8667ZM16.6667 16.8667C17.6938 16.8667 18.5333 16.0271 18.5333 15H18.1333C18.1333 15.8062 17.4729 16.4667 16.6667 16.4667V16.8667ZM18.5333 15V4.99999H18.1333V15H18.5333ZM18.5333 4.99999C18.5333 3.97287 17.6938 3.13332 16.6667 3.13332V3.53332C17.4729 3.53332 18.1333 4.19378 18.1333 4.99999H18.5333ZM16.6667 3.13332H14.025V3.53332H16.6667V3.13332ZM14.1726 3.19831L12.6476 1.53165L12.3524 1.80167L13.8774 3.46833L14.1726 3.19831ZM10 7.69999C11.2645 7.69999 12.3 8.73545 12.3 9.99999H12.7C12.7 8.51453 11.4855 7.29999 10 7.29999V7.69999ZM12.3 9.99999C12.3 11.2645 11.2645 12.3 10 12.3V12.7C11.4855 12.7 12.7 11.4854 12.7 9.99999H12.3ZM10 12.3C8.73545 12.3 7.7 11.2645 7.7 9.99999H7.3C7.3 11.4854 8.51454 12.7 10 12.7V12.3ZM7.7 9.99999C7.7 8.73545 8.73545 7.69999 10 7.69999V7.29999C8.51454 7.29999 7.3 8.51453 7.3 9.99999H7.7ZM10 5.63332C7.58954 5.63332 5.63333 7.58953 5.63333 9.99999H6.03333C6.03333 7.81045 7.81045 6.03332 10 6.03332V5.63332ZM5.63333 9.99999C5.63333 12.4104 7.58954 14.3667 10 14.3667V13.9667C7.81045 13.9667 6.03333 12.1895 6.03333 9.99999H5.63333ZM10 14.3667C12.4105 14.3667 14.3667 12.4104 14.3667 9.99999H13.9667C13.9667 12.1895 12.1895 13.9667 10 13.9667V14.3667ZM14.3667 9.99999C14.3667 7.58953 12.4105 5.63332 10 5.63332V6.03332C12.1895 6.03332 13.9667 7.81045 13.9667 9.99999H14.3667Z"
                fill={Colors.theme.primary}
                mask="url(#path-1-inside-1_1_564)"
            />
        </Svg>
    );
};

const style = StyleSheet.create({
    root: {
        width: 'auto',
        height: 'auto'
    }
});

export default Camera;
