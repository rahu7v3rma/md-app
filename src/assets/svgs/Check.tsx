import React, { FunctionComponent } from 'react';
import { Path, Rect, Svg } from 'react-native-svg';

import { Colors } from '@/theme/colors';

type Props = {
    width?: number;
    height?: number;
    color?: string;
};

const CheckIcon: FunctionComponent<Props> = ({
    width = 97,
    height = 96,
    color = Colors.extras.white
}: Props) => {
    return (
        <Svg width={width} height={height} viewBox="0 0 97 96" fill="none">
            <Rect
                x={0.5}
                width={width}
                height={height}
                rx={32}
                fill={Colors.theme.primary}
            />
            <Path
                d="M62.5 38l-20 20-8-8"
                stroke={color}
                strokeWidth={4}
                strokeLinecap="round"
                strokeLinejoin="round"
            />
        </Svg>
    );
};

export default CheckIcon;
