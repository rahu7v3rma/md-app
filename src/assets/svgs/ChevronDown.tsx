import React, { FunctionComponent } from 'react';
import { StyleSheet } from 'react-native';
import { Path, Svg } from 'react-native-svg';

import { Colors } from '@/theme/colors';

type Props = {
    width?: number;
    height?: number;
};

const ChevronDown: FunctionComponent<Props> = ({
    width = 12,
    height = 8
}: Props) => {
    return (
        <Svg
            width={width}
            height={height}
            viewBox="0 0 12 8"
            fill="none"
            style={style.root}
        >
            <Path
                d="M11 1.5L6 6.5L1 1.5"
                stroke={Colors.theme.primary}
                stroke-width="2"
                stroke-linecap="round"
                stroke-linejoin="round"
            />
        </Svg>
    );
};

const style = StyleSheet.create({
    root: {
        width: 'auto',
        height: 'auto'
    }
});

export default ChevronDown;
