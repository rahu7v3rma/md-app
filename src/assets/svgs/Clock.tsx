import React, { FunctionComponent } from 'react';
import { Path, Svg } from 'react-native-svg';

type Props = {
    width?: number;
    height?: number;
    color?: string;
};

const ClockIcon: FunctionComponent<Props> = ({
    width = 11,
    height = 12,
    color = '#6E51D0'
}: Props) => {
    return (
        <Svg width={width} height={height} viewBox="0 0 11 12" fill="none">
            <Path
                d="M0.776001 6.76999C0.776001 4.16599 2.896 2.04599 5.5 2.04599C8.104 2.04599 10.224 4.171 10.224 6.776C10.224 9.38 8.104 11.5 5.5 11.5C2.896 11.5 0.776001 9.37499 0.776001 6.76999ZM5.5 7.354C5.923 7.354 6.271 7.00601 6.271 6.58301V3.66699C6.271 3.24299 5.923 2.896 5.5 2.896C5.077 2.896 4.729 3.24299 4.729 3.66699V6.58301C4.729 7.00601 5.077 7.354 5.5 7.354Z"
                fill={color}
                stroke={color}
                stroke-width="0.666667"
            />
            <Path
                d="M7.18597 0.678986H3.81403C3.76503 0.678986 3.72699 0.64201 3.72699 0.59201C3.72699 0.53701 3.77103 0.5 3.81403 0.5H7.18597C7.23497 0.5 7.27197 0.537006 7.27197 0.587006C7.27197 0.642006 7.22897 0.678986 7.18597 0.678986Z"
                fill={color}
                stroke={color}
                stroke-width="0.666667"
            />
        </Svg>
    );
};

export default ClockIcon;
