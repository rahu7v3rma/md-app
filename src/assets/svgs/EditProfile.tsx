import * as React from 'react';
import Svg, { Path } from 'react-native-svg';

type Props = {
    width?: number;
    height?: number;
    color?: string;
};

const EditProfile: React.FunctionComponent<Props> = ({
    width = 18,
    height = 18,
    color = '#fff'
}: Props) => {
    return (
        <Svg width={width} height={height} viewBox="0 0 20 20" fill="none">
            <Path
                d="M7.93 8.806l3.516-3.515a2.307 2.307 0 113.263 3.263l-3.515 3.516a11.925 11.925 0 01-5.54 3.136l-.68.17a.289.289 0 01-.35-.35l.17-.68a11.924 11.924 0 013.137-5.54z"
                fill={color}
                fillOpacity={0.24}
            />
            <Path
                d="M11.446 5.291l.53.53-.53-.53zM7.93 8.806l.53.53-.53-.53zm6.778-.252l-.53-.53.53.53zm-3.515 3.516l-.53-.53.53.53zm-5.54 3.136l-.182-.728.182.728zm-.68.17l.183.727-.182-.727zm-.35-.35l.728.181-.728-.182zm.17-.68l.728.182-.728-.182zm6.121-9.585L7.4 8.276l1.06 1.06 3.516-3.514-1.06-1.061zm3.263 3.263l-3.515 3.515 1.061 1.06 3.515-3.514-1.06-1.06zm-8.706 6.454l-.68.17.365 1.455.68-.17-.365-1.455zm-.12.73l.17-.68-1.456-.364-.17.68 1.456.363zm-.56-.56a.461.461 0 01.56.56l-1.455-.365c-.19.761.499 1.45 1.26 1.26l-.364-1.455zm5.871-3.109a11.173 11.173 0 01-5.19 2.94l.363 1.455a12.674 12.674 0 005.888-3.334l-1.06-1.06zM7.4 8.276a12.674 12.674 0 00-3.334 5.888l1.456.364A11.174 11.174 0 018.46 9.337L7.4 8.277zm5.677-4.41c-.81 0-1.588.322-2.162.895l1.06 1.06a1.557 1.557 0 011.102-.456v-1.5zm1.558 3.057c0 .413-.165.809-.457 1.101l1.061 1.06a3.057 3.057 0 00.896-2.161h-1.5zm1.5 0a3.057 3.057 0 00-3.058-3.058v1.5c.86 0 1.558.698 1.558 1.558h1.5zM13.8 8.513c-.431.144-1.098-.052-1.68-.634-.582-.582-.778-1.249-.634-1.68l-1.423-.475c-.4 1.2.22 2.438.997 3.215.777.778 2.014 1.397 3.215.997L13.8 8.513z"
                fill={color}
            />
        </Svg>
    );
};

export default EditProfile;
