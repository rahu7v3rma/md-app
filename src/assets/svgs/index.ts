import ArrowNext from './ArrowNext';
import BackIcon from './Back';
import CheckIcon from './Check';
import ChevronDown from './ChevronDown';
import ChoosePhoto from './ChoosePhoto';
import ClockIcon from './Clock';
import CrossIcon from './Cross';
import EditProfile from './EditProfile';
import EmailIcon from './EmailIcon';
import EyeIcon from './Eye';
import EyeOpenIcon from './EyeOpen';
import Heart from './Heart';
import HighFive from './HighFive';
import Like from './Like';
import LoginLogoIcon from './LoginLogo';
import LogoutIcon from './logout';
import Muscle from './Muscle';
import NoData from './NoData';
import BellIcon from './notification';
import PlayIcon from './Play';
import Plus from './Plus';
import ProfileIcon from './ProfileIcon';
import SadFace from './SadFace';
import SearchIcon from './Search';
import Setting from './Setting';
import SliderMarker from './SliderMarker';
import SmileyFace from './SmileyFace';
import TakePhoto from './TakePhoto';
import WolfIcon from './Wolf';
import WolfIconNotes from './WolficonNotes';
import WolfIconDance from './WolfiocnDance';

export {
    BackIcon,
    BellIcon,
    CheckIcon,
    ChevronDown,
    CrossIcon,
    EditProfile,
    EmailIcon,
    EyeIcon,
    EyeOpenIcon,
    Heart,
    HighFive,
    Like,
    LoginLogoIcon,
    LogoutIcon,
    Muscle,
    PlayIcon,
    ProfileIcon,
    SadFace,
    SliderMarker,
    SmileyFace,
    WolfIcon,
    Plus,
    NoData,
    ClockIcon,
    SearchIcon,
    WolfIconNotes,
    WolfIconDance,
    ArrowNext,
    ChoosePhoto,
    TakePhoto,
    Setting
};
