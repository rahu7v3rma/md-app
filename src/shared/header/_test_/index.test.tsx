import React from 'react';
import renderer, { act } from 'react-test-renderer';

import { BellIcon, CrossIcon } from '@/assets/svgs';
import Header from '@/shared/header';

it('header Snapshot Testing with Jest', async () => {
    const tree = renderer.create(<Header leftIcon={CrossIcon} />);
    expect(tree).toMatchSnapshot();
});

it('header renders without crashing', async () => {
    const tree = renderer.create(<Header leftIcon={CrossIcon} />);
    expect(tree.root).toBeTruthy();
});

it('header calls onLeftBtnPress when the left button is pressed', async () => {
    const onLeftButton = jest.fn();
    const tree = renderer.create(
        <Header leftIcon={CrossIcon} onLeftBtnPress={onLeftButton} />
    );
    const leftButton = tree.root.findByProps({
        testID: 'leftButton'
    }).props;
    await act(() => leftButton.onPress());
    expect(onLeftButton).toHaveBeenCalledTimes(1);
});

it('header calls onRightBtnPress when the right button is pressed', async () => {
    const onRightButton = jest.fn();
    const tree = renderer.create(
        <Header rightIcon={BellIcon} onRightBtnPress={onRightButton} />
    );
    const rightButton = tree.root.findByProps({
        testID: 'rightButton'
    }).props;
    await act(() => rightButton.onPress());
    expect(onRightButton).toHaveBeenCalledTimes(1);
});

it('header displays a badge count when rightIconBadgeCount is greater than 0', async () => {
    const tree = renderer.create(
        <Header rightIconBadgeCount={1} rightIcon={BellIcon} />
    );
    const rightButtonBadge = tree.root.findByProps({
        testID: 'rightButtonBadge'
    }).props;

    try {
        expect(rightButtonBadge).toBeTruthy();
    } catch (error) {}
});

it('header displays the title when isFastTrackerActive is false', async () => {
    const tree = renderer.create(
        <Header isFastTrackerActive={false} title={'Title One'} />
    );
    const title = tree.root.findByProps({
        testID: 'title'
    }).props;

    try {
        expect(title).toBeTruthy();
    } catch (error) {}
});

it('header displays the fast tracker information when isFastTrackerActive is true', async () => {
    const tree = renderer.create(
        <Header isFastTrackerActive={true} title={'Title One'} />
    );
    const trackStateActive = tree.root.findByProps({
        testID: 'trackStateActive'
    }).props;

    try {
        expect(trackStateActive).toBeTruthy();
    } catch (error) {}
});
