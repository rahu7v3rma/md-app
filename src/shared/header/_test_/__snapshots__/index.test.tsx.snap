// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header Snapshot Testing with Jest 1`] = `
<View>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": true,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "alignItems": "center",
        "backgroundColor": "#FFFFFF",
        "flexDirection": "row",
        "justifyContent": "center",
        "opacity": 1,
        "paddingBottom": 22,
        "paddingHorizontal": 19,
        "paddingTop": 10,
      }
    }
  >
    <View
      style={
        {
          "alignItems": "flex-start",
          "width": "20%",
        }
      }
    >
      <View
        accessibilityState={
          {
            "busy": undefined,
            "checked": undefined,
            "disabled": undefined,
            "expanded": undefined,
            "selected": undefined,
          }
        }
        accessibilityValue={
          {
            "max": undefined,
            "min": undefined,
            "now": undefined,
            "text": undefined,
          }
        }
        accessible={true}
        collapsable={false}
        focusable={false}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          {
            "alignItems": "center",
            "backgroundColor": "#D3E29F40",
            "borderRadius": 100,
            "height": 48,
            "justifyContent": "center",
            "margin": 0,
            "opacity": 1,
            "paddingHorizontal": 15.5,
            "paddingVertical": 24,
            "shadowColor": "",
            "shadowOffset": {
              "height": 0,
              "width": 0,
            },
            "shadowOpacity": 0,
            "width": 48,
          }
        }
        testID="leftButton"
      >
        <RNSVGSvgView
          align="xMidYMid"
          bbHeight={24}
          bbWidth={24}
          fill="none"
          focusable={false}
          height={24}
          meetOrSlice={0}
          minX={0}
          minY={0}
          style={
            [
              {
                "backgroundColor": "transparent",
                "borderWidth": 0,
              },
              {
                "height": "auto",
                "width": "auto",
              },
              {
                "flex": 0,
                "height": 24,
                "width": 24,
              },
            ]
          }
          vbHeight={24}
          vbWidth={24}
          width={24}
        >
          <RNSVGGroup
            fill={null}
            propList={
              [
                "fill",
              ]
            }
          >
            <RNSVGPath
              d="M18.7071 6.70711C19.0976 6.31658 19.0976 5.68342 18.7071 5.29289C18.3166 4.90237 17.6834 4.90237 17.2929 5.29289L18.7071 6.70711ZM5.29289 17.2929C4.90237 17.6834 4.90237 18.3166 5.29289 18.7071C5.68342 19.0976 6.31658 19.0976 6.70711 18.7071L5.29289 17.2929ZM17.2929 18.7071C17.6834 19.0976 18.3166 19.0976 18.7071 18.7071C19.0976 18.3166 19.0976 17.6834 18.7071 17.2929L17.2929 18.7071ZM6.70711 5.29289C6.31658 4.90237 5.68342 4.90237 5.29289 5.29289C4.90237 5.68342 4.90237 6.31658 5.29289 6.70711L6.70711 5.29289ZM17.2929 5.29289L5.29289 17.2929L6.70711 18.7071L18.7071 6.70711L17.2929 5.29289ZM18.7071 17.2929L6.70711 5.29289L5.29289 6.70711L17.2929 18.7071L18.7071 17.2929Z"
              fill={4280752721}
              propList={
                [
                  "fill",
                ]
              }
            />
          </RNSVGGroup>
        </RNSVGSvgView>
      </View>
    </View>
    <View
      style={
        {
          "alignItems": "center",
          "width": "60%",
        }
      }
    >
      <Text
        adjustsFontSizeToFit={true}
        style={
          [
            {
              "fontFamily": "Poppins",
            },
            undefined,
            {
              "color": "#72996B",
              "fontSize": 16,
              "fontWeight": "600",
              "lineHeight": 24,
            },
          ]
        }
        testID="title"
      />
    </View>
    <View
      style={
        {
          "alignItems": "flex-end",
          "width": "20%",
        }
      }
    />
  </View>
  <View
    style={false}
  >
    <View
      style={false}
    />
  </View>
</View>
`;
